src/core/core.ts

```diff
@@ -46,15 +46,20 @@ const executeWithContext = function (
   resources: MilkResource[],
   context: Map<string, any>
 ): Promise<any> {
+  const logPath = (resource: MilkResource) => {
+    console.log(`path: ${resource.metadata.path}`);
```

Comment:
Would it be better to use a logging library like Winston?

Reply:
makes sense, will mark as a follow up change